### ISSUE TITLE
改动centernet的head

### ISSUE THREAD

**Issue by YYL-SCAR** [bubbliiiing/centernet-pytorch#56]
B导你好 我在一篇定会论文（Parsing Table Structures in the Wild ）看到他们把centernet的预测宽高改为预测四个顶点，但并没有公开源码。我想复现这个改动，请问是要在对每个顶点再生成一个热力图吗

**Comment by bubbliiiing** [comment]
额0 0…可以这样，但是我感觉没啥用啊为什么

**Comment by YYL-SCAR** [comment]
因为这个论文检测的是扭曲的自然场景下的单元格，预测四边形四个顶点更接近原始形状，而且后续他还要用这些顶点做很复杂的后处理构造另一个损失函数。不过我又看了下他新的一篇论文（也是centernet），他是用一个convhead回归出四个角点来代替预测宽高。我这样写可以吗：
 #预测四个顶点
        self.bbox_head = nn.Sequential(
            nn.Conv2d(64, channel,
                      kernel_size=3, padding=1, bias=False),
            nn.BatchNorm2d(64, momentum=bn_momentum),
            nn.ReLU(inplace=True),
            nn.Conv2d(channel, 8,
                      kernel_size=1, stride=1, padding=0))
